docs/_layouts/default.html

```diff
@@ -23,16 +23,25 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Apache Flink: {{ page.title }}</title>
-    <link rel="shortcut icon" href="{{ site.baseurl }}favicon.ico" type="image/x-icon">
-    <link rel="icon" href="{{ site.baseurl }}favicon.ico" type="image/x-icon">
-    <link rel="stylesheet" href="{{ site.baseurl }}css/bootstrap.css">
-    <link rel="stylesheet" href="{{ site.baseurl }}css/bootstrap-lumen-custom.css">
-    <link rel="stylesheet" href="{{ site.baseurl }}css/syntax.css">
-    <link rel="stylesheet" href="{{ site.baseurl }}css/custom.css">
-    <link href="{{ site.baseurl }}css/main/main.css" rel="stylesheet">
+    <link rel="shortcut icon" href="{{ site.baseurl }}/favicon.ico" type="image/x-icon">
+    <link rel="icon" href="{{ site.baseurl }}/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/bootstrap.css">
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/bootstrap-lumen-custom.css">
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/syntax.css">
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/custom.css">
+    <link href="{{ site.baseurl }}/css/main/main.css" rel="stylesheet">
```

Comment:
This will break the loading of stylesheets on our website. {{ site.baseurl }} is actually not set in the config. Thus the path "/css" will load the css path of http://ci.apache.org